CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ find_package(
 )
 # </Boost>
 
+# <yaml-cpp>
+find_package(yaml-cpp CONFIG REQUIRED)
+# <yaml-cpp>
```

Comment:
maybe put the `find_package(catch2)` in here as well so `src/third_party` is only used for vendored libs

Reply:
Nevermind here - I see the `find_package` is still there for the "real" catch2 impl and we just have the "main" file now. Fine to leave as-is!